rust/cubesql/cubesql/src/compile/engine/df/wrapper.rs

```diff
@@ -1500,7 +1500,21 @@ impl CubeScanWrapperNode {
                             f.map(|f| format!("{}", f)).unwrap_or("NULL".to_string()),
                             sql_query,
                         ),
-                        // ScalarValue::Decimal128(_, _, _) => {}
+                        ScalarValue::Decimal128(x, precision, scale) => {
+                            // In Postgres, NUMERIC or DECIMAL scale can be negative.  But it's unsigned, here.
+                            let scale: usize = scale;
+
+                            (
+                                x.map(|x| {
+                                    let number =
+                                        crate::compile::rewrite::rules::filters::Decimal::new(x)
```

Comment:
This almost seems like trolling...

Where would you put that function's behavior?